exec/cnex/lib/process_win32.c

```diff
@@ -9,6 +9,30 @@
 #include "nstring.h"
 
 
+struct ThreadInfo {
+    HANDLE pHandle;
+    Cell *buffer;
+};
+
+static DWORD WINAPI reader_thread(LPVOID lpParam)
+{
+    struct ThreadInfo *ti = lpParam;
+
+    while (ti->pHandle != INVALID_HANDLE_VALUE) {
```

Comment:
You don't have to keep checking that the handle is valid. Nothing is going to set it invalid.

Reply:
Changed to a `for (;;)` loop.